examples/xml-access/xml_access.description

```diff
@@ -1,2 +1,2 @@
 // XML elements can contain child XML items. Ballerina allows you to access these
-// child elements conveniently using field-access and index-access expressions.
+// child elements conveniently using xml step expressions and xml filter expressions.
```

Comment:
xml -> XML. Use `xml` if referring to the XML type in Ballerina. If referring to XML in general, use XML (all caps, not highlighted as code)